messages/role_annoucement.md

```diff
@@ -0,0 +1,15 @@
+Text for announcement to astropy-dev mailing list
+
+The Coco proposes to [select as appropriate]
+- add/remove the following roles to the list of roles on the Astropy project:
+- add/remove the following people for role XXX in the Astropy project:
+
+See https://www.astropy.org/team for a list of current roles. Following APE 0 [1] we leave a two week long comment period for this proposal where you can comment publicly on the PR [2] or privately to the Coco (coordinators@astropy.org or on slack). If there are no objections after two weeks, the Coco will merge the PR [3].
+
+Yours,
```

Comment:
I always find "Yours" weird. This is not a love letter. Is it just me?
```suggestion
Sincerely,
```

Reply:
I think there is a difference between BE and AE (and Australian, South African, Indian, etc. English for that matter). It's how we learned in school (BE) to end letters that are not formal enough for "yours sincerely", for example between friends or equals as opposed to from student to professor or from seller to buyer in a contract negotiation.  A short google suggests at least 15 or so more variants... (https://www.google.com/search?client=firefox-b-1-d&q=yours+letter+ending#ip=1) from "yours truely" (BE) and "yours faithfully" (AE) to "cheers". I'd not use a simple "yours" in a love letter... but I'm not going to post here what I would say in that case. Bottom line, I'm fine with either.